src/components/sidebar.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export const Sidebar = () => {
+  return (
+    <aside
+      id="default-sidebar"
+      className="fixed top-0 left-0 z-40 w-64 h-screen"
+      aria-label="Sidebar"
+    >
+      <div className="h-full px-3 py-4 overflow-y-auto bg-gray-50 dark:bg-gray-800">
+        <ul className="hover:font-semibold">
+          <li>
```

Comment:
Her ville det nok være en god ide at lave et SideBarItem components som kan genbruges i din sidebar

Reply:
også kan jeg anbefale at bruge heroicons for icons, i stedet for at hardcode svg ind i koden. Der findes react biblioteker til dette (og de bruger svg under the hood, så du mister ikke noget niceness)